### DISCUSSION TITLE
Attach glossary term to attribute of an entity through Purview REST API

### DISCUSSION THREAD

**Discussion by nupoor01nawathey** [Azure/azure-rest-api-specs#29670] [Q&A] [answered]
I'm unable to find the exact API request to attach glossary term to attribute of an entity for Purview REST API. Just for experiment tried below and working. It would really help if this API details could be added in the official documentation.

Steps to extract the API request call. 
1. I updated the glossary term at attribute level from the Purview portal.
2. Then used GET API request for the entity guid to extract the response.
3. Then use part of the JSON response which has key "meanings" as a POST request for updating glossary from the backend.

I could see this working as expected.

**Comment by mikeharder** [comment]
Can you please clarify your question so it can be routed to the correct team?  Is your question for the ARM team, the Azure SDK team, or the Purview team?

**Reply by nupoor01nawathey** [reply]
The suggestion is for the Purview team. The sample REST API for attaching glossary term to attribute level of an entity is missing from the github examples and from official documentation as well. Upon checking with MS purview support team, they advised to add a request on github.

**Reply by mikeharder** [reply, marked as answer]
I recommend double-checking with the Purview team exactly how they would like to track your feature request.  Is "azure-rest-api-specs" even the right repo?  If so, you probably want to open an "issue" rather than a "discussion".

https://github.com/Azure/azure-rest-api-specs/issues
